argocd/app_plane/alloy.yaml

```
@@ -0,0 +1,317 @@
+              configMap:
+                content: |-
+                  prometheus.remote_write "in_cluster" {
+                    endpoint {
+                      url = "http://vmsingle-vm-victoria-metrics-k8s-stack.observability.svc.cluster.local:8429/prometheus/api/v1/write"
+                    }
+                  }
+
+                  prometheus.remote_write "ctrl_plane" {
+                    endpoint {
+                      url = "https://vmauth.observability.dev.internal.falkordb.cloud/prometheus/api/v1/write"
+                      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)